modules/ossm-directing-egress-traffic-through-a-gateway-using-istio-apis.adoc

```diff
@@ -0,0 +1,226 @@
+// This procedure is used in the following assembly:
+// * gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-directing-egress-traffic-through-a-gateway-using-istio-apis_{context}"]
+= Directing egress traffic through a gateway using Istio APIs
+:context: ossm-directing-egress-traffic-through-a-gateway-using-istio-apis
+
+Follow this procedure to use {istio} APIs to direct outbound HTTP traffic through a gateway that was installed using gateway injection.
```

Comment:
Not critical for this PR's merge, but in the future should avoid self-referential language like "follow this procedure"

Reply:
done